cmd/boulder-va/main.go

```diff
@@ -18,6 +19,17 @@ import (
 type Config struct {
 	VA struct {
 		vaConfig.Common
+
+		// RVATLSClient is a distinct and separate cmd.TLSConfig used by the
+		// boulder-va client for communication with a remoteva server. Typically
+		// this block should be configured identically to the "tls"
+		// cmd.TLSConfig or not at all which will cause the boulder-va client to
+		// use the "tls" configuration. It should typically be used in
+		// conjunction with the remoteva "VerifyGRPCClientCertIfGiven" boolean
+		// which may also need to be adjusted depending on infrastructure
+		// characteristics of a remoteva deployment.
+		RVATLSClient cmd.TLSConfig `validate:"structonly"`
```

Comment:
I don't understand why this half of the PR is necessary. The VA only has remoteva gRPC clients -- it doesn't reach out to the SA or RA or anything else. So it doesn't need two different client configs.